config/environments/qa.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
Why is this file being added?

Reply:
The resque:hotswap cap step was failing because there was no environment file matching qa.  Once I added it the task completed. So i assumed it was needed, but I defer to you if it isn't needed.

Reply:
This is only used when the RAILS_ENV=qa.  The RAILS_ENV for this environment should be "production"

Reply:
I think the kurma-robots are set up incorrectly in puppet.  It should be similar to common-accessioning: https://github.com/sul-dlss/puppet/blob/production/hieradata/project/common-accessioning-qa.eyaml#L22-L23

Reply:
So this line should be set to 'production': https://github.com/sul-dlss/puppet/blob/4082eaa2cd26351f1a70e23512ff60c173f561e9/hieradata/node/kurma-robots-qa-01.stanford.edu.eyaml#L41 ?

Reply:
Yes, that is correct.

Reply:
I've updated the PR to remove the qa file, and made the change in puppet described above to point it to production pool

Reply:
Thank you all for your help.